tasks/48_follow-the-spy/follow-the-spy.js

```diff
@@ -0,0 +1,19 @@
+export function followTheSpy(arrayOfRoutes) {
+  let mapOfRoutes = {};
+  const start = arrayOfRoutes[0][0];
+  let path = [start];
+
+  arrayOfRoutes.forEach(([from, to]) => {
+    mapOfRoutes[from] = to;
+  });
+
+  while (mapOfRoutes[lastElement(path)]) {
+    path.push(mapOfRoutes[lastElement(path)]);
+  }
+
+  return path.join(', ');
+
+  function lastElement(array) {
```

Comment:
Move lastElement() to the top of the module.
Also, consider better name like "getLastElement()".
In many naming conventions functions start with the verbs.